app/models/apple/episode.rb

```diff
@@ -258,6 +255,30 @@ def episode_create_parameters
       }
     end
 
+    def update_episode_bridge_params
```

Comment:
I think some of this parameter handling code is a bit too nested, (long method call chains with similar names).

I have a few ideas on how to fix this up in the future.  I think we can flatten some of these calls, maybe use some blocks to DRY things up.